test/unit/controller/edit_merchant_details_controller/post_test.js

```diff
@@ -186,6 +190,153 @@ describe('edit merchant details controller - post', () => {
       })
     })
   })
+  describe('when the update merchant details call has invalid telephone number', () => {
+    before(done => {
+      response = serviceFixtures.validUpdateMerchantDetailsResponse(serviceRoles[0].service.merchant_details).getPlain()
+      adminusersMock.put(`${SERVICE_RESOURCE}/${EXTERNAL_SERVICE_ID}/merchant-details`)
+        .reply(200, response)
+      let userInSession = mockSession.getUser({
+        external_id: 'exsfjpwoi34op23i4',
+        service_roles: serviceRoles
+      })
+      session = {
+        csrfSecret: '123',
+        12345: {refunded_amount: 5},
+        passport: {
+          user: userInSession
+        },
+        secondFactor: 'totp',
+        last_url: 'last_url',
+        version: 0
+      }
+      let app = mockSession.createAppWithSession(getApp(), session)
+      supertest(app)
+        .post(formattedPathFor(paths.merchantDetails.update, EXTERNAL_SERVICE_ID))
+        .set('Content-Type', 'application/x-www-form-urlencoded')
+        .send({
+          'merchant-name': 'new-name',
+          'telephone-number': 'call me maybe',
+          'address-line1': 'new-line1',
+          'address-city': 'new-city',
+          'address-postcode': 'new-postcode',
+          'address-country': 'AR',
+          'merchant-email': 'dd-merchant@example.com',
+          csrfToken: csrf().create('123')
+        })
+        .end((err, res) => {
+          response = res
+          done(err)
+        })
+    })
+    it(`should redirect back to the page`, () => {
+      expect(response.statusCode).to.equal(302)
+      expect(response.headers.location).to.equal(formattedPathFor(paths.merchantDetails.index, EXTERNAL_SERVICE_ID))
+    })
+    it(`should set errors in the session`, () => {
+      expect(session.pageData.editMerchantDetails.success).to.be.false // eslint-disable-line
+      expect(session.pageData.editMerchantDetails.errors).to.deep.equal({
+        'telephone-number': true
+      })
+    })
+  })
+  describe('when the update merchant details call has invalid email', () => {
+    before(done => {
+      response = serviceFixtures.validUpdateMerchantDetailsResponse(serviceRoles[0].service.merchant_details).getPlain()
+      adminusersMock.put(`${SERVICE_RESOURCE}/${EXTERNAL_SERVICE_ID}/merchant-details`)
+        .reply(200, response)
+      let userInSession = mockSession.getUser({
+        external_id: 'exsfjpwoi34op23i4',
+        service_roles: serviceRoles
+      })
+      session = {
+        csrfSecret: '123',
+        12345: {refunded_amount: 5},
+        passport: {
+          user: userInSession
+        },
+        secondFactor: 'totp',
+        last_url: 'last_url',
+        version: 0
+      }
+      let app = mockSession.createAppWithSession(getApp(), session)
```

Comment:
Minor: we should use `const` by default, then if we need to reassign the variable we should use `let` and we should not use `var`. I know the whole file was already using `let` but it's good to update when we touch new files. Thanks